docs/mllib-frequent-pattern-mining.md

```diff
@@ -77,22 +79,60 @@ that stores the frequent itemsets with their frequencies.
 import java.util.List;
 
 import com.google.common.base.Joiner;
+import com.google.common.collect.Lists;
```

Comment:
Guava utils are not necessary here. Could you help verify the following?

1. The `Joiner` usage can be removed. `println("[" + Joiner.on(",").join(itemset.javaItems()) + "]")` is the same as `println(itemset.javaItems())`.
2. `Lists.newArrayList` could be replaced by Java's `Arrays.asList`.

Then we can remove Guava dependency from the example code.